NEWS

```diff
@@ -1,5 +1,10 @@
 This file lists noteworthy changes in SaunaFS.
 
+ * SaunaFS (4.5.1) (2024-09-16)
+ - Fix critical issue with uraft assigning floating IP address to all nodes.
```

Comment:
If Ganesha improvements are included in the pull request, they shoud also be included in the `NEWS` and in the `changelog` files.

Reply:
I was thinking about leaving them out (since they are only for the testing framework), but you're probably right, should include them for consistency with the commits at least.

However, small comment: You should not use feat for test because otherwise our current tool will bump the minor version, which we don't want (since it's not changing/adding functionality)

Reply:
I understand, I did not know.
I used **feat** because the commit adds a feature at the tests level :)
But it's not a problem, I will not do it in the future.